api_key_stamper/README.md

```diff
@@ -0,0 +1,51 @@
+# tkhq_api_key_stamper
+
+This crate contains structs and utilities to work with P-256 keys, which [Turnkey](https://docs.turnkey.com/) uses as a primary way of authentication.
+
+## Creating a new P-256 API key
+
+```rust
+use tkhq_api_key_stamper::TurnkeyP256ApiKey
+
+let api_key = TurnkeyP256ApiKey::generate();
+```
+
+## Loading API keys from env
+
+If you keep API keys in env vars, load it with `from_bytes` or `from_strings`:
+
+```rust
+use std::env;
+use tkhq_api_key_stamper::TurnkeyP256ApiKey;
+
+// Assuming the env var is a hex-encoded string
```

Comment:
Might be worth refreshing the repo README as well. :)

Reply:
Yes, good catch!